scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
+set -e
+
+# Get the path to the script directory
+SCRIPT_DIR="$(eval dirname $0)"
+
+# Get the SHASUM of the tarball
+ZIP_FILE="${SCRIPT_DIR}/apollo.tar.gz"
+SHASUM="$(eval /usr/bin/shasum -a 256 ${ZIP_FILE})"
```

Comment:
It's worth noting that the output of in the full path to the tarball (i.e. `123abc /Users/you/path/you/have/zippy-dee-doo-dah`), which means the path is baked into the `.shasum` and might be less portable (it would re-extract if you just moved the parent folder to another folder).  If it was worth avoiding that, this could be:

```suggestion
SHASUM="$(/usr/bin/shasum -a 256 ${ZIP_FILE})"
```

Reply:
The problem is that this is being executed from a different directory - so it needs the full rather than the relative path (and will be different based on where Carthage, Cocoapods, and Swift Package manager check things out). I agree that's not ideal, but adding a relative path that works for all those different dependency managers isn't really feasible.

Reply:
the `eval` is gone, at least